examples/handlers/async_handlers/README.md

```diff
@@ -0,0 +1,88 @@
+# Async Request Handlers
+
+A slightly more involved example than [Request Data](../request_data), which
+does work asyncronously and resolves the response future with the result.
+
+## Running
+
+From the `examples/handlers/async_handlers` directory:
+
+```
+Terminal 1:
+$ cargo run
+   Compiling handlers/async_handlers (file:///.../examples/handlers/async_handlers)
+    Finished dev [unoptimized + debuginfo] target(s) in 4.26 secs
+     Running `../handlers/async_handlers`
+Listening for requests at http://127.0.0.1:7878
+Method: Get
```

Comment:
TODO: this is just copy-paste from the other example. I actually want to do something like:
```
for i in `seq 50` ; do curl 'http://127.0.0.1:7878/?length='$i ; echo =$i; done
```